src/Neo.CLI/CLI/MainService.Tools.cs

```diff
@@ -268,7 +349,27 @@ private string ClearHexString(string hexString)
         }
 
         /// <summary>
-        /// Converts an address to its corresponding scripthash
+        /// Fix for Base64 strings containing unicode
+        /// Input  DCECbzTesnBofh/Xng1SofChKkBC7jhVmLxCN1vk\u002B49xa2pBVuezJw==
+        /// Output DCECbzTesnBofh/Xng1SofChKkBC7jhVmLxCN1vk+49xa2pBVuezJw==
+        /// </summary>
+        /// <param name="str">Base64 strings containing unicode</param>
+        /// <returns>Correct Base64 string</returns>
+        public static string Base64Fixed(string str)
+        {
+            MatchCollection mc = Regex.Matches(str, @"\\u([\w]{2})([\w]{2})", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
TryCatch is faster and cheaper for checking `Base64` encoding. Just `TryCatch` `Convert.FromBase64String` and look for `FormatException`